internal/eval/eval_test.go

```diff
@@ -2520,7 +2521,7 @@ func testEvalLLEN(t *testing.T, store *dstore.Store) {
 				evalSET([]string{"EXISTING_KEY", "mock_value"}, store)
 			},
 			input:  []string{"EXISTING_KEY"},
-			output: []byte("-ERR Existing key has wrong Dice type\r\n"),
+			output: []byte("-Existing key has wrong Dice type\r\n"),
```

Comment:
Is this change in line with Redis output?

Reply:
> Hi, this return can to be updated. Currently the message in dice is: ![image](https://private-user-images.githubusercontent.com/32174554/372313022-0de95ba7-e744-418a-88f2-06a330eb0d8d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwNDE2NTIsIm5iZiI6MTcyODA0MTM1MiwicGF0aCI6Ii8zMjE3NDU1NC8zNzIzMTMwMjItMGRlOTViYTctZTc0NC00MThhLTg4ZjItMDZhMzMwZWIwZDhkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDExMjkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE2ZjI0YjQ1ZWIyZWUxZjQ0MWU1MThkMDcwYjYxOTNkODliMDkxNWEwYTI4MGU0ODZhYzdhNTc0ZGIzMjk5YzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.o3YWSSgOBm8iSLN409y_mahmYQVdHEJeZTSQggR4OHU) But in redis: ![image](https://private-user-images.githubusercontent.com/32174554/372313064-08e8c8c1-7230-41d8-9d36-165c6417cb05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwNDE2NTIsIm5iZiI6MTcyODA0MTM1MiwicGF0aCI6Ii8zMjE3NDU1NC8zNzIzMTMwNjQtMDhlOGM4YzEtNzIzMC00MWQ4LTlkMzYtMTY1YzY0MTdjYjA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDExMjkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1YjE0MjFlOGI2ZTI3OTQ1YzhiNDQxNDZkNWQwMWVlYjAxNmZjMGM0NzM1ZWFlOWM2Mjg3ODA5M2YxZjA0YTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FPuI12LmyeTVlS8dV-tmHBTZbpxhg1_nFCvKo2qPYe8) I understand we have to replace dice with redis, but we have an extra ERR in the message which can be removed if we modify the `object.AssertTypeAndEncoding` with the following change:
> 
> ```
> func AssertTypeAndEncoding(typeEncoding, expectedType, expectedEncoding uint8) []byte {
> 	if err := AssertType(typeEncoding, expectedType); err != nil {
> 		return diceerrors.NewErrWithMessage(diceerrors.WrongKeyTypeErr)
> 	}
> 	if err := AssertEncoding(typeEncoding, expectedEncoding); err != nil {
> 		return diceerrors.NewErrWithMessage(diceerrors.WrongKeyTypeErr)
> 	}
> 	return nil
> }
> ```
> 
> and add the following error in `errors.go` file:
> 
> ```
> 	WrongKeyTypeErr        = "-Existing key has wrong Dice type"
> ```
> 
> How do you feel about this suggestion.

Yeah It's same as the Dice output

Reply:
Ok I'll revert it back to original